app.py

```diff
@@ -6,10 +6,20 @@ def add(a,b):
 def sub(a,b):
     return a-b
 
+def add_fact(a,b):
+	return fact(add(a,b))
 
+def fact(a):
+	if a < 0: 
+		return "Factorial of Negative Number!"
+	if a == 0 or a==1:
```

Comment:
It is better to use elif here.

The first form if-if-if tests all conditions, whereas the second if-elif-else tests only as many as needed: if it finds one condition that is True , it stops and doesn't evaluate the rest. In other words: if-elif-else is used when the conditions are mutually exclusive.

Reply:
Thanks! Got it.